docs/using.rst

```diff
@@ -3,8 +3,8 @@ Using the RAUC hawkbit Updater
 
 .. _authentication-section:
 
-Authentication
---------------
```

Comment:
Please leave this section as is and add the sections below as subsections.

Reply:
done